GeeksCoreLibrary/Components/Account/Interfaces/IAccountsService.cs

```diff
@@ -80,5 +79,60 @@ public interface IAccountsService
         /// <param name="includePermissions">Optional: Whether to include all permissions that each role has. Default is <see langword="false"/>.</param>
         /// <returns></returns>
         Task<List<RoleModel>> GetUserRolesAsync(ulong userId, bool includePermissions = false);
+
+        /// <summary>
+        /// NOTE: This method was moved from Accounts component and it should NOT be exposed to users.
+        /// Automatically logs in the user via ID. This function should never be made available publicly, only for internal usage to login after creating an account for example.
+        /// </summary>
+        /// <param name="userId">The ID of the user to login.</param>
+        /// <param name="mainUserId">The ID of the main user, if the user is logging in with a sub account.</param>
+        /// <param name="role">Used to set a custom role for the user separate of the Wiser role system</param>
+        /// <param name="extraDataForReplacements"></param>
+        /// <param name="settings"></param>
+        Task AutoLoginUserAsync(ulong userId, ulong mainUserId, string role,
```

Comment:
Please keep the function on 1 line or 1 line per parameter, instead of a split in the middle.